src/meta-srv/src/metasrv.rs

```diff
@@ -79,6 +79,13 @@ pub struct MetaSrvOptions {
     pub wal: MetaSrvWalConfig,
     pub export_metrics: ExportMetricsOption,
     pub store_key_prefix: String,
+    /// The max operations per txn
+    ///
+    /// If using etcd as the store, this value should be less than or equal to the
+    /// `--max-txn-ops` option value of etcd. If using another store, this option
+    /// can be ignored.
+    /// For more detail, see: https://etcd.io/docs/v3.5/op-guide/configuration/
+    pub max_txn_ops: usize,
```

Comment:
I wonder if we add a `etcd_` prefix? Otherwise it may confuse users that the option doesn't work for other store - it's a understanding burden.

Reply:
I want this option can be spread to all stores in the future, because a big `txn` is dangerous, it may blocks other operations. I left a TODO here.